app/javascript/packages/webauthn/webauthn-input-element.ts

```diff
@@ -13,12 +13,14 @@ export class WebauthnInputElement extends HTMLElement {
     return this.hasAttribute('show-unsupported-passkey');
   }
 
-  toggleVisibleIfPasskeySupported() {
+  async toggleVisibleIfPasskeySupported() {
     if (!this.hasAttribute('hidden')) {
       return;
     }
-
-    if (isWebauthnPasskeySupported()) {
+    const isPublicKeyCredentialSupported =
```

Comment:
For specs, maybe we can look at what was removed in 243efcd337d560d6ab1c22c636fa87d2fb2fdd59 as prior art for how to stub in specs.